views/js/qtiCreator/plugins/menu/preview.js

```diff
@@ -70,17 +84,18 @@ define([
                 title: __('Preview the item'),
                 text : __('Preview'),
                 cssClass: 'preview-trigger'
-            })).on('click', function previewHandler(e){
-                $(document).trigger('open-preview.qti-item');
-
-                e.preventDefault();
-
-                self.disable();
-
-                itemCreator.trigger('preview', itemCreator.getItem().data('uri'));
-
-                self.enable();
-            });
+            })).on('click', previewHandler.bind(this));
```

Comment:
Can it be subscribed to an item-add event?

In that case it will be even better.

1. An empty test
2. An author adds an item
3. The preview button immediately becomes active
4. The author clicks on the button
5. Save/cancel dialog prompted

And the other way around with removing items of course.

Reply:
No item-add event exist in items. That click event was setted before, I add a if conditional just for avoid event triggered when disabled (posible new bug in that component)

![image](https://user-images.githubusercontent.com/34692207/95342593-e2ada400-08b7-11eb-9cf5-d26281f676db.png)

Or maybe you mean a different thing? Because I reproduced your steps and no issues

Reply:
![ezgif-5-55ce33b97ff2](https://user-images.githubusercontent.com/2943256/95427634-264fee80-0948-11eb-9f0a-1b2010ab2ca8.gif)

I mean, making the button available immediately after adding an item would've been more user-friendly.

Reply:
@wazelin But that will be a different issue, this is only for fix preview in authoring items, not authoring test.

I can create

Reply:
OMG, you're right.
I got a bit too far there.

No need to do it here.